sdk/tables/Azure.Data.Tables/samples/README.md

```diff
@@ -22,8 +22,5 @@ You can find samples for each of this main functions below.
 To get started you'll need an Azure Tables endpoint and credentials. See Azure Tables Client Library [Readme][README] for more information and instructions.
 
 - [Create/delete tables](https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/tables/Azure.Data.Tables/samples/Sample1CreateDeleteTables.md)
-- [Create/delete entities](https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/tables/Azure.Data.Tables/samples/Sample2CreateDeleteEntities.md)
```

Comment:
you could either have this in and after you merge it will work. or if you are more comfortable you could wait, merge, and then create a PR with the links.

Reply:
The Github checks didn't work since the links didn't exist. :( Is there a way around that?

Reply:
ohh sad... that is a new functionality they added,,, so I guess no. Ignore my comment :(